### PR TITLE
[jsk_fetch_startup] Fix description report-light-on

### PR DESCRIPTION
**Pull request by tkmtnt7000**
Thanks to https://github.com/knorth55/jsk_robot/pull/343#issuecomment-1283283835.

Fix description in `report-light-on`.

### CODE REVIEW

**Review by tkmtnt7000 — COMMENTED** [review]
Memo:
Use https://github.com/jsk-ros-pkg/jsk_roseus/pull/724

**Comment by knorth55** [comment]
I update https://github.com/jsk-ros-pkg/jsk_roseus/pull/724 and merged into `knorth55/jsk_roseus@fetch15` branch.
https://github.com/jsk-ros-pkg/jsk_roseus/pull/715

Now, you can use `userdata` instead of `mydata`.
Also, with `before-hook-func`, you dont need these two lines. https://github.com/jsk-ros-pkg/jsk_robot/pull/1679/files#diff-db470e8365f9cf73b75d39c7822ddc1513068c80cf3e12dd4a3127cba08d2863R771-R772

File: jsk_fetch_robot/jsk_fetch_startup/euslisp/go-to-kitchen.l [hunk 1]
```diff
@@ -29,7 +29,11 @@
                                      (success-go-to-kitchen . nil)
                                      (success-go-to-trashcan . nil)
                                      (success-auto-dock . nil))
-                              :hz 2.0)))
+                              :hz 2.0
+                              :before-hook-func
+                              '(lambda ()
+                                (set-alist 'description "" 'userdata)
+                                (set-alist 'image "" 'userdata)))))
```
Comment (knorth55) [inline jsk_fetch_robot/jsk_fetch_startup/euslisp/go-to-kitchen.l:36]: can you try like this with the latest `fetch15` branch.
I chnaged to pass userdata as argument.
[jsk-ros-pkg/jsk_roseus@`e5e739c` (#715)](https://github.com/jsk-ros-pkg/jsk_roseus/pull/715/commits/e5e739c9fccd45a6a3c047043ef21400f1884ae1)

```suggestion
                              '(lambda (userdata)
                                (set-alist 'description "" userdata)
                                (set-alist 'image "" userdata)))))
```

**Comment by sktometometo** [comment]
@tkmtnt7000 Is this PR still valid? If so, could you rebase this PR to current `develop/fetch`?

**Comment by tkmtnt7000** [comment]
I think [d69d852](https://github.com/jsk-ros-pkg/jsk_robot/pull/1679/commits/d69d852533fdb1f36fd34474b06bc3e286a07167) and [8ddcf92](https://github.com/jsk-ros-pkg/jsk_robot/pull/1679/commits/8ddcf92563cf989b4c3c81d981ca4440e6efbb8c) have not merged yet, but this feature is not used now. So I'll close this PR.

**Pull request closed without merging**